scripts/miniapps.py

```diff
@@ -157,6 +157,13 @@ def submitJobs(self, run_dir, debug=False):
             stdout=PIPE,
             stderr=PIPE,
         )
+
+        raw_stdout, raw_stderr = process.communicate()
+        print(raw_stdout, end="")
+        print(raw_stderr, end="")
+        if process.returncode != 0:
+            print(f"Job submission FAILED.")
```

Comment:
In the case that one is submitting locally without slurm, will this mean that if there's an actual miniapp failure, the script will stop submitting jobs? That might not be the desired behaviour, but then I never use that so I'm not sure what the desired behaviour is.

@biddisco's input would be very useful here.

Reply:
No, actually we just report the output/error but we don't stop in any case the execution.

I never used it locally, so I might be missing something. In the local case, the return code depends on the last command executed by the bash script, and just if the last one fails the "FAILED" message is printed, but no "FAILED" printed if any of the "middle" runs fails.

Reply:
Ah, good point, I misread the code... In that case, probably no concern here from my side.